db_init_script.py

```diff
@@ -24,17 +24,19 @@ def main():
 
     base = "BTC"
     coin = "ETH"
-    spread = ".1"
-    merkato = Merkato(configuration, coin, base, spread)
+    spread = .1
```

Comment:
i think the proper fix is to make sure that what uses this runs float() on it regardless.
who knows what format the DB returns.

Reply:
Decimal*

Reply:
the db has it saved as a float, it will return it as such

Reply:
@nasaWelder

Reply:
We should attempt to use and store Decimal whenever possible